docs/configuration/upstream/propose_downstream.md

```diff
@@ -17,6 +23,8 @@ happen in Fedora dist-git - no builds. A pull request is created as a result.
 * **dist_git_branches** - a (list of) branches in dist-git where packit should work (defaults to `main` which represents _Fedora Rawhide_).
   You can also use the [aliases provided by Packit](/docs/configuration#aliases)
   to not need to change the config file when the new system version is released.
+* **pkg_tool** - `fedpkg` (default) or `centpkg`. Needs to be specified if you want to sync release to CentOS Stream,
```

Comment:
Unless you've opened PR for this, we don't support it. The issue is that the `LocalProject` is initialized with the `pkg_tool`, so you need to have it specified **before** the job. We probably “hack” it somehow, but that introduces another problem of `LocalProject` depending on the job config during initialization.

> **NOTE**
> I _THINK_ it doesn't work :D

Reply:
aah ok, I wasn't aware of this, will change it